src/pages/BrowseSchemes.svelte

```diff
@@ -17,6 +19,11 @@
   import { bbox } from "../maplibre_helpers";
   import { gjScheme, map } from "../stores";
 
+  onMount(() => {
+    // For govuk components. Must happen here.
```

Comment:
Following https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#import-javascript-using-a-bundler, I originally tried calling it in `browse.html` outside of the Svelte stuff entirely, but the accordion example had no JS. No console errors or indication that something went wrong. If we call this in the component script outside of `onMount`, it also doesn't seem to work. So it must rely on part of the DOM being set up?